string topic_name
string msg_type #optional, will be Image
---
sensor_msgs/Image image
